apps/els_dap/src/els_dap_breakpoints.erl

```diff
@@ -108,32 +108,38 @@ get_function_breaks(Module, Breaks) ->
 get_line_breaks(Module, Breaks) ->
     case Breaks of
         #{Module := #{line := Lines}} -> Lines;
-        _ -> []
+        _ -> #{}
     end.
 
 -spec do_line_breakpoints(
     node(),
     module(),
     #{line() => line_breaks()},
-    breakpoints()
+    breakpoints(),
+    boolean()
 ) ->
     breakpoints().
-do_line_breakpoints(Node, Module, LineBreakPoints, Breaks) ->
-    maps:map(
-        fun(Line, _) -> els_dap_rpc:break(Node, Module, Line) end,
-        LineBreakPoints
-    ),
+do_line_breakpoints(Node, Module, LineBreakPoints, Breaks, Set) ->
+    case Set of
```

Comment:
In which case do we get `Set=false`?

Reply:
When the module does not exist, e.g. the debugged node cannot load the module.